pkgs/development/python-modules/geoip2/default.nix

```diff
@@ -30,6 +30,8 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  disabledTests = lib.optional (pythonAtLeast "3.11") "TestAsyncClient";
```

Comment:
For me there was also a failure on Python 3.10, see https://github.com/NixOS/nixpkgs/pull/217295/commits/e2be01564248989a4dd2192f8410b3e62fd80f9b